R/chromote_session.R

```diff
@@ -10,29 +10,50 @@ ChromoteSession <- R6Class(
   lock_objects = FALSE,
   cloneable = FALSE,
   public = list(
+    #' @description Create a new `ChromoteSession` object.
```

Comment:
I think maybe you need to enable markdown in the DESCRIPTION file?

```
Roxygen: list(markdown = TRUE)
```

https://roxygen2.r-lib.org/articles/markdown.html

Reply:
That did it, thank you. Should I include the new/updated `.Rd`s in this PR?

Newest Roxygen now also complains a lot about missing R6 documentation, but I figure we can leave that to a new PR.